src/dapi/src/cmd/par_def1.h

```diff
@@ -1,9 +1,9 @@
 /*
  ***********************************************************************
  *
- *                           Copyright �
- *	  Copyright � 2000-2001 Force Computers Inc., a Solectron company. All rights reserved.
- *    � Digital Equipment Corporation 1995, 1998. All rights reserved.
+ *                           Copyright �

```

Comment:
File has no changes except whitespace and the broken copyright. Maybe not include it in this PR.